hw/ip_templates/pwrmgr/rtl/pwrmgr_fsm.sv.tpl

```diff
@@ -291,8 +290,7 @@ module pwrmgr_fsm import pwrmgr_pkg::*; import pwrmgr_reg_pkg::*;(
 
   mubi4_t rom_intg_chk_done;
   mubi4_t rom_intg_chk_good;
-  assign rom_intg_chk_done = mubi4_or_hi(mubi4_and_hi(rom_intg_chk_dis, rom_ctrl_done_i),
-                                         rom_ctrl_done_i);
+  assign rom_intg_chk_done = mubi4_or_hi(rom_intg_chk_dis, rom_ctrl_done_i);
```

Comment:
Are there any risks to releasing the pwrmgr FSM faster? I note that until rom_ctrl is done with the check, the rom_ctrl's internal FSM keeps control of the bus, effectively preventing execution by the CPU.

Reply:
Ha, that's funny: I was wondering exactly the same thing! If the point is just that we don't require the check to *pass* in RMA, I think that's already done correctly with the calculation for `rom_intg_chk_good` on the next line.

So maybe this commit should just tidy up the slightly dubious code, add a comment explaining what's going on, but leave the functionality as before :-D

But it's possible I don't know something at the system level?